app/main.py

```diff
@@ -2,0 +1,10 @@
+def copy_file(command: str) -> None:
+    if len(command.split()) != 3:
+        raise ValueError("cp command can take only 2 arguments")
+    cp_command, file, new_file = command.split()
+    if cp_command != "cp":
+        raise ValueError("cp command not found")
+
+    if file != new_file:
+        with open(file, "r") as input_file, open(new_file, "w") as output_file:
```

Comment:
Code Style: The variable names `input_file` and `output_file` are not descriptive enough. Consider using more meaningful names, like `source_file` and `destination_file`.